.gitignore

```diff
@@ -17,4 +17,6 @@ packages/discovery/mock_local_storage
 .giga
 .cursor
 .DS_Store
-CLAUDE.md
\ No newline at end of file
+CLAUDE.md
+packages/discovery/scratch/
```

Comment:
what is it?

Reply:
I'm not sure but I think it's generated any time tests are run

Reply:
https://github.com/waku-org/js-waku/blob/49974402254a793a7ea1865515813c5ec94e0cc5/packages/discovery/src/local-peer-cache/index.spec.ts#L24

Reply:
🤦 then let's keep it